_specs/ecip-1069.md

```diff
@@ -0,0 +1,78 @@
+---
+lang: en
+ecip: 1069
+title: Account Versioning Extension for CREATE and CREATE2
+author: Wei Tang (@sorpaas)
+discussions-to: https://github.com/ethereumclassic/ECIPs/issues/169
+status: Draft
+type: Standards Track
+category: Core
+created: 2019-06-24
```

Comment:
Same point here as others regarding date of submission of an ECIP specifically vs the original EIP. Because you're adding the Apache License as part of the submission, I think adding the correct date for the PR submission makes sense.